script/fix_documentation.py

```diff
@@ -0,0 +1,67 @@
+"""Small script to fix various issues with the documentation. Used by pre-commit."""
+# fmt: off
```

Comment:
Why add the ``# fmt off`` ? Is black ruining the file ? 😄

Reply:
Yeah, I'm not a huge fan of black, personally. If this file gets bigger over time, you can remove the comment. For now I think it looks better with it being turned off.

Reply:
The thing is, pylint is using black not only to minimize ``git diff`` but also so we do not have to argue about formatting. If we start disabling black on some files and not the others we're opening a can of subjective worms. I'm going to address this in another MR.

Reply:
I would have changed it already if you have asked earlier 😅
Although I would prefer it being turned off, I understand your concerns.

Reply:
Haha don't worry, I first thought "not very important, let's merge". Then I thought of the number of maintainer reply per month it would generate :D